connection.go

```diff
@@ -575,16 +588,21 @@ func (c *Connection) readLoop() {
 
 func (c *Connection) readResponseLoop() {
 	for {
+		readTimeoutTimer := time.NewTimer(c.Opts.ReadTimeout)
```

Comment:
same here, we can `Reset` the timer instead of creating and stopping it in each loop.